### PR TITLE
 Improve bookmark link and fix typo

### PR DESCRIPTION
**Pull request by verdantpeas**
- ~~Generated bookmarklet from [here](https://mrcoles.com/bookmarklet/)~~ 😞 
- fix typo in "title" property 

~~Note: new bookmark link as fixed typo~~

### CODE REVIEW

**Comment by vandyliu** [comment]
Sorry for the PR race :(

**Comment by verdantpeas** [comment]
> Sorry for the PR race :(

LOOOL IT WAS YOU?!

**Comment by ianmah** [comment]
Good catch and very nice this is way easier for ppl to use 👍 
Could you regenerate the link in the README to incorporate the changes from #1 :)

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Create query test cases quickly. For UBC CPSC 310 students. [demo](https://www.y
...
...2C%203000%2C%20()%20%3D%3E%20%7BsetTimeout(()%20%3D%3E%20%7Bconst%20queryLog%20%3D%20console.logs%5B1%5D%3Bconst%20resultLog%20%3D%20console.logs%5B2%5D%3Bconsole.logs%20%3D%20%5B%5D%3Bconst%20query%20%3D%20queryLog%5B0%5D%3Bconst%20result%20%3D%20resultLog%20%26%26%20resultLog%5B1%5D%3Bconst%20obj%20%3D%20result%20%3F%7Btitle%3A%20%22generated%20by%20https%3A%2F%2Fgithub.com%2Fianmah%2F310-bookmarklet.%20Pls%20gib%20star%22%2Cquery%2CisQueryValid%3A%20true%2C...result%7D%20%3A%20%7Btitle%3A%20%22generated%20by%20https%3A%2F%2Fgithub.com%2Fianmah%2F310-bookmarklet.%20Pls%20gib%20star%22%2Cquery%2CisQueryValid%3A%20false%2Cresult%3A%20%22InsightError%22%7D%3Bconst%20json%20%3D%20JSON.stringify(obj%2C%20null%2C%204)%3BcopyToClipboard(json)%3Bconsole.logs%20%3D%20%5B%5D%3B%7D%2C%20420)%3B%7D)%3B%7D%3Bconst%20parent%20%3D%20document.getElementById(%22submit-container%22)%3Bconst%20button%20%3D%20document.createElement(%22a%22)%3Bbutton.innerHTML%20%3D%20%22Copy%20Test%20File%22%3Bbutton.id%20%3D%20%22generate-button%22%3Bbutton.className%20%3D%20%22btn%22%3Bbutton.href%20%3D%20%22%23%22%3Bbutton.onclick%20%3D%20onClick%3Bparent.appendChild(button)%3B%7D%7D)()%7D)()">310-bookmarklet</a>
```
Comment (vandyliu) [inline README.md]: This link isn't clickable for me, when I look at it here: https://github.com/joshflchan/310-bookmarklet/tree/improve-bookmark-link-and-fix-typo

**Review by ianmah — COMMENTED** [review]
Good catch and very nice this is way easier for ppl to use 👍
Could you also regenerate the link in the README to incorporate the changes from #1 :)

File: README.md (hunk 1 above)
Comment (verdantpeas) [reply README.md]: Looks like it is only working for me locally 
image

Comment (ianmah) [reply README.md]: Have you tried the `[text](link)` syntax?

Comment (verdantpeas) [reply README.md]: It's doomed https://stackoverflow.com/questions/14951321/how-to-display-html-content-in-github-readme-md 
No `href` allowed in github md

**Pull request closed without merging**